tests/functional/spec/features/placeholder/placeholder_default_value.spec.js

```diff
@@ -0,0 +1,33 @@
+import EmployeesNumberBlockPage from "../../../generated_pages/placeholder_default_value/employees-number-block.page";
+import EmployeesTrainingBlockPage from "../../../generated_pages/placeholder_default_value/employees-training-block.page";
+import EmployeesNumberDefaultInterstitialPage from "../../../generated_pages/placeholder_default_value/employees-number-default-interstitial.page";
+
+describe("Placeholder default value check", () => {
+  beforeEach("Load the survey", () => {
+    browser.openQuestionnaire("test_placeholder_default_value.json");
+  });
+
+  it("Given a question with default answer, When I do not enter any number and click submit, Then the interstitial page shows default employees number as 0", () => {
+    $(EmployeesNumberBlockPage.submit()).click();
+    expect($("#main-content > p").getText()).to.contain("The total number of employees confirmed are 0 (default value)");
```

Comment:
Do these not have a selector?. We should avoid arbitrary selectors in spec files.

Reply:
It is not arbitrary , the selector is` main-content`

Reply:
Resolved since it is an existing issue as we dont have a selector for it, and won't hold up this.